src/palette/elements/Button/ButtonV2.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { action } from "@storybook/addon-actions"
+import { storiesOf } from "@storybook/react-native"
+import { LinkIcon } from "palette"
+import { ButtonSize, ButtonV2 as Button, ButtonVariant } from "palette/elements/Button/ButtonV2"
+import React from "react"
+import { DList, List } from "storybook/helpers"
+
+const sizes: ButtonSize[] = ["small", "medium", "large"]
+
+const variants: ButtonVariant[] = [
+  "primaryBlack",
+  "primaryWhite",
+  "secondaryGray",
+  "secondaryOutline",
+  "secondaryOutlineWarning",
+  "noOutline",
+]
+
+storiesOf("ButtonV2", module)
```

Comment:
```suggestion
storiesOf("Theme/ButtonV2", module)
```
By adding this, it makes a little folder. It might be cool to have it that way, and then we can add Screens etc too 😮

Reply:
Seems like nesting is [not yet implemented](https://github.com/storybookjs/react-native/issues/12).

Reply:
Ah, on the browser it works, but ondevice UI is not supporting it. I guess it could still be ok, but I'm open to suggestions. We could also keep it as is?